filesystem.go

```diff
@@ -58,6 +65,29 @@ func New(fs billy.Filesystem, path string) SivaFS {
 	}
 }
 
+// NewFilesystem creates an entire filesystem using siva as the main backend,
+// but supplying unsupported functionality using as a temporal files backend
+// the main filesystem
```

Comment:
Explain that the temporal files are created in the `fs` filesystem, instead of in memory, for instance.